static/app/components/eventOrGroupHeader.tsx

```diff
@@ -84,32 +84,18 @@ function EventOrGroupHeader({
       );
     }
 
-    // If we have passed in a custom event ID, use it; otherwise use default
-    const finalEventId = eventId ?? latestEventId;
-
     return (
       <TitleWithLink
         {...commonEleProps}
-        to={{
```

Comment:
Moved all this to a util function so it can be used in `stream/group.tsx` as well